cmake/OpenCVDetectFindGflags.cmake

```diff
@@ -420,7 +420,7 @@ if (NOT GFLAGS_FOUND)
     set(CALLERS_CMAKE_FIND_LIBRARY_PREFIXES "${CMAKE_FIND_LIBRARY_PREFIXES}")
     # The empty string in this list is important, it represents the case when
     # the libraries have no prefix (shared libraries / DLLs).
-    set(CMAKE_FIND_LIBRARY_PREFIXES "lib" "" "${CMAKE_FIND_LIBRARY_PREFIXES}")
+    # LaurentBerger For Opencv PR 7700 set(CMAKE_FIND_LIBRARY_PREFIXES "lib" "" "${CMAKE_FIND_LIBRARY_PREFIXES}")
```

Comment:
This code is for `GFLAGS_RESET_FIND_LIBRARY_PREFIX` / `GLOG_RESET_FIND_LIBRARY_PREFIX` macros only.
Changes near `set(CMAKE_FIND_LIBRARY_PREFIXES ...)` should be reverted.